### PR TITLE
change methods names

### PR DESCRIPTION
**Pull request by SandyEma9**

### CODE REVIEW

**Comment by SandyEma9** [comment]
@robert-altom @RobertPoienar @gomboskriszta15

**Review by RobertPoienar — COMMENTED** [review]
Depending on what kind of release are we deciding to do (major or minor) this PR is correct or incorrect. Because it breaks backward compatibility if we decide to do a minor release then it would great if you mark the methods with small letters as deprecated and create new methods with correct syntax. If we decide to release a major version aka PRO then this is correct

File: Assets/AltTester/AltDriver/Common/AltObject.cs [hunk 1]
```diff
@@ -47,17 +47,28 @@ public AltObject GetParent()
             CommHandler.SleepFor(CommHandler.GetDelayAfterCommand());
             return altObject;
         }
-
+        public AltObject getParent()
```
Comment (RobertPoienar) [inline Assets/AltTester/AltDriver/Common/AltObject.cs:50]: Mark the methods Obsolete so users will know to use the other method. Here is how you can do it: https://stackoverflow.com/questions/1759352/how-to-mark-a-method-as-obsolete-or-deprecated

File: Assets/AltTester/AltDriver/Common/AltObject.cs [hunk 2]
```diff
@@ -47,17 +47,28 @@ public AltObject GetParent()
...
+            var altObject = new AltFindObject(CommHandler, By.PATH, "//*[@id=" + this.id + "]/..", By.NAME, "", true).Execute();
+            CommHandler.SleepFor(CommHandler.GetDelayAfterCommand());
+            return altObject;
+        }
         public AltVector2 GetScreenPosition()
         {
             return new AltVector2(x, y);
         }
-
+        public AltVector2 getScreenPosition()
```
Comment (RobertPoienar) [inline Assets/AltTester/AltDriver/Common/AltObject.cs:60]: Also this one needs to be obsolete

File: Assets/AltTester/AltDriver/Common/AltObject.cs [hunk 3]
```diff
@@ -47,17 +47,28 @@ public AltObject GetParent()
...
+        {
+            return new AltVector2(x, y);
+        }
         public AltVector3 GetWorldPosition()
         {
             return new AltVector3(worldX, worldY, worldZ);
         }
-
+        public AltVector3 getWorldPosition()
```
Comment (RobertPoienar) [inline Assets/AltTester/AltDriver/Common/AltObject.cs:68]: and this one

**Review by RobertPoienar — CHANGES_REQUESTED** [review]
Mark the old methods as obsolete

**Comment by SandyEma9** [comment]
@RobertPoienar @robert-altom @ReichEdvin

**Review by gomboskriszta15 — APPROVED** [review]
👍

**Pull request merged**